src/views/home/devices.js

```diff
@@ -0,0 +1,509 @@
+// This list of common devices was put together from various online sources by me, Jan Keromnes.
+//
+// The device features are:
+// - name: The device brand and model(s).
+// - width: The viewport width.
+// - height: The viewport height.
+// - pixelRatio: The screen's pixel ratio (e.g. HiDPI > 1).
+// - userAgent: The device's UserAgent string on the web.
+// - touch: Whether the device's screen is touch-enabled.
+//
+// The different types of devices are:
+// - phones (sources: [1] + various)
+// - tablets (sources: [1] + various)
+// - notebooks (sources: [1])
+// - televisions (sources: various)
+// - watches (sources: various)
```

Comment:
extremely stoked that this includes watches

Reply:
:joy: :joy: